libvmaf/src/libvmaf.rc.c

```diff
@@ -360,7 +361,7 @@ int vmaf_score_pooled(VmafContext *vmaf, VmafModel *model,
 
 const char *vmaf_version(void)
 {
-    return "RELEASE_CANDIDATE";
+    return VMAF_VERSION;
```

Comment:
How is this versioning different from a major.minor.fix convention?

Reply:
This prints out just an abbreviated hash, and nothing similar to semver

```none
./libvmaf/build/tools/vmaf_rc --version
311ea4c
```

Reply:
Hi, yes this corresponds to the abbreviated commit hash rather than the semantic version, which allows us to point to the exact commit id used.